### PR TITLE
Add -b param for ignoring blank spaces

### LINKED ISSUE #7: Parameter -b

### PR DESCRIPTION
**Pull request by yhabteab**
resolves #7

### CODE REVIEW

File: sort.php [hunk 1]
```diff
@@ -66,6 +66,10 @@ function bubbleSort($sort_array, $checkEquality)
     $makeUnique = array_values(array_unique($makeUnique));
 }
 
+if (key_exists("b", $options)) {
+    $makeUnique = array_map("ltrim", $makeUnique);
+}
+
```
Comment (Al2Klimov) [inline sort.php:68]: A bit too late.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]

File: sort.php (hunk 1 above)
Comment (yhabteab) [reply sort.php:68]: I will then have to call ``file ()`` twice.

Comment (Al2Klimov) [reply sort.php:68]: You don't even need file() here.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
* [ ] `git reset --soft HEAD~$(git rev-list --count $(git merge-base HEAD master)..HEAD^)`
* [ ] `git commit --amend --no-edit`
* [ ] `git push -f origin feature/sort-#7`

**Comment by yhabteab** [comment]
##Testprotocol #7

Test for the parameter ``--ignore-leading-blanks`` or in abbreviation ``-b``.

The content of the in.txt file looks like this.
You can also see a few empty lines at the end.
```
4
9
8
1
6



```
If you execute the commands with the -b parameter, the output looks like this..
``php sort.php -b in.txt`` AND ``php sort.php -b < in.txt``  output:
```
1                                1
2                                2
4                                4
6                                6
```

See you next time!

Yonas :+1:

**Review by Al2Klimov — APPROVED** [review]

**Pull request merged**